tests/test_freshdesk_start_date.py

```diff
@@ -0,0 +1,166 @@
+        if len(second_sync_bonus) > 0:
+            print("Found stream(s): {} in second sync. Add to test_streams?".format(second_sync_bonus))
+
+        for stream in test_streams:
+            with self.subTest(stream=stream):
+
+                if stream in self.second_sync_empty:
+                    print("No sync 2 data to compare for stream: {}, start_date obeyed".format(stream))
+
+                    continue
+
+                # expected values
+                expected_primary_keys = self.expected_primary_keys()[stream]
+                expected_start_date_1 = self.timedelta_formatted(self.start_date_1, days=0)  # Insight buffer format saved
+                expected_start_date_2 = self.timedelta_formatted(self.start_date_2, days=0)
+
+                # collect information for assertions from syncs 1 & 2 base on expected values
+                record_count_sync_1 = record_count_by_stream_1.get(stream, 0)
+                record_count_sync_2 = record_count_by_stream_2.get(stream, 0)
+                primary_keys_list_1 = [tuple(message.get('data').get(expected_pk) for expected_pk in expected_primary_keys)
+                                       for message in synced_records_1.get(stream).get('messages')
+                                       if message.get('action') == 'upsert']
+                primary_keys_list_2 = [tuple(message.get('data').get(expected_pk) for expected_pk in expected_primary_keys)
+                                       for message in synced_records_2.get(stream).get('messages')
+                                       if message.get('action') == 'upsert']
+                primary_keys_sync_1 = set(primary_keys_list_1)
+                primary_keys_sync_2 = set(primary_keys_list_2)
+
+                if stream in obey_start_date_streams:
+                    print("Stream {} obeys start_date".format(stream))
+                    # collect information specific to incremental streams from syncs 1 & 2
+                    expected_replication_key = next(iter(self.expected_replication_keys().get(stream)))
+                    replication_dates_1 =[row.get('data').get(expected_replication_key) for row in
+                                          synced_records_1.get(stream, {'messages': []}).get('messages', [])
+                                          if row.get('data')]
+                    replication_dates_2 =[row.get('data').get(expected_replication_key) for row in
+                                          synced_records_2.get(stream, {'messages': []}).get('messages', [])
+                                          if row.get('data')]
+
+                    # Verify replication key is greater or equal to start_date for sync 1
+                    for replication_date in replication_dates_1:
+                        self.assertGreaterEqual(
+                            self.parse_date(replication_date), self.parse_date(expected_start_date_1),
```

Comment:
Should only need to parse if the bookmarks are save in a different format than start_date. Both should be ISO-8601 (actualy RFC --something)